extensions/wikia/AdEngine/js/spec/video/player/playwire/playwire.spec.js

```diff
@@ -1,5 +1,5 @@
 /*global describe, it, expect, modules, beforeEach, spyOn*/
-describe('ext.wikia.adEngine.video.playwire', function () {
+describe('ext.wikia.adEngine.video.player.playwire.playwire', function () {
```

Comment:
This will be conflicted: https://github.com/Wikia/app/pull/12104/files#diff-1d57a09ec87deb67f155fcc6cf2da82fR2